packages/@aws-cdk/aws-elasticloadbalancingv2/lib/nlb/network-load-balancer.ts

```diff
@@ -38,6 +38,14 @@ export interface NetworkLoadBalancerAttributes {
    * @default - When not provided, LB cannot be used as Route53 Alias target.
    */
   readonly loadBalancerDnsName?: string;
+
+  /**
+   * The VPC to associate with the load balancer.
+   *
+   * @default - When not provided, listeners cannot be created on imported load
+   * balancers.
+   */
+  readonly loadBalancerVpc?: ec2.IVpc;
```

Comment:
On the application load balancer this prop is called `vpc`, why is it called `loadBalancerVpc` here? Please make both the same. `vpc` will suffice.